fi.csc.Elmer.yaml

```diff
@@ -0,0 +1,380 @@
+app-id: fi.csc.Elmer
+runtime: org.kde.Platform
+sdk: org.kde.Sdk
+runtime-version: 5.15-22.08
+command: ElmerGUI
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
```

Comment:
I see tk is being build. If tk is actually used, then X11 needs to be available at all times.